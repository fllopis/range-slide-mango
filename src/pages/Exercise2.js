import React from "react";

const Exercise2 = () => {
  return (
    <div>
      <h1> Exercise2 </h1>
    </div>
  );
};

export default Exercise2;