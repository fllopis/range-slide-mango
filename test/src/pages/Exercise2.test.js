import React from 'react'
import Exercise2 from '../../../src/pages/Exercise2'
import { render } from '@testing-library/react';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';

describe('Testing page Exercise2', () => {
    
    const history = createMemoryHistory();
    
    test('Should load the page Exercise2', () => {
        
        const wrapper = render(
            <Router location={history.location} navigator={history}> <Exercise2 /> </Router>);
            expect(wrapper).toMatchSnapshot();
        }
    );
})