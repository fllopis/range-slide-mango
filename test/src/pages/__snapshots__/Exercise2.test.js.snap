// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing page Exercise2 Should load the page Exercise2 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
       
      <div
        class="mt-4"
      >
        <h2>
          Fixed range picker
        </h2>
        <p>
          A fixed range picker, is mount through an array of values. You can drag any dot and this will put in the correct position.
        </p>
        <p>
          You can't edit the input values for this example.
        </p>
        <section>
          <div
            class="row mt-5"
          >
            <div
              class="col-4 offset-4"
            >
              <div
                class=""
              >
                <div
                  class="d-flex justify-content-center rangePickerSlide_container"
                >
                  <div
                    class="rangePicker__Input minInput"
                  >
                    <span
                      class="actualPositionTxt"
                    >
                      0
                    </span>
                  </div>
                  <div
                    actual-value="0"
                    class="rangePicker__dot dot__left"
                    id="dot-left"
                    max-value="0"
                    min-value="0"
                    style="left: 0%;"
                  />
                  <div
                    class="rangePicker__line"
                  />
                  <div
                    actual-value="0"
                    class="rangePicker__dot dot__right"
                    id="dot-right"
                    max-value="0"
                    min-value="0"
                    style="left: 100%;"
                  />
                  <div
                    class="rangePicker__Input maxInput"
                  >
                    <span
                      class="actualPositionTxt"
                    >
                      0
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </section>
      </div>
       
    </div>
  </body>,
  "container": <div>
     
    <div
      class="mt-4"
    >
      <h2>
        Fixed range picker
      </h2>
      <p>
        A fixed range picker, is mount through an array of values. You can drag any dot and this will put in the correct position.
      </p>
      <p>
        You can't edit the input values for this example.
      </p>
      <section>
        <div
          class="row mt-5"
        >
          <div
            class="col-4 offset-4"
          >
            <div
              class=""
            >
              <div
                class="d-flex justify-content-center rangePickerSlide_container"
              >
                <div
                  class="rangePicker__Input minInput"
                >
                  <span
                    class="actualPositionTxt"
                  >
                    0
                  </span>
                </div>
                <div
                  actual-value="0"
                  class="rangePicker__dot dot__left"
                  id="dot-left"
                  max-value="0"
                  min-value="0"
                  style="left: 0%;"
                />
                <div
                  class="rangePicker__line"
                />
                <div
                  actual-value="0"
                  class="rangePicker__dot dot__right"
                  id="dot-right"
                  max-value="0"
                  min-value="0"
                  style="left: 100%;"
                />
                <div
                  class="rangePicker__Input maxInput"
                >
                  <span
                    class="actualPositionTxt"
                  >
                    0
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
     
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
